--- leptonlib-1.52/src/endiantest.c.orig	2008-02-20 20:09:43.000000000 +0000
+++ leptonlib-1.52/src/endiantest.c	2008-02-20 20:13:09.000000000 +0000
@@ -1,11 +1,12 @@
 /*  endiantest.c  */
 
 #include <stdio.h>
+#include <sys/types.h>
 
 int main()
 {
 /* fprintf(stderr, "doing the test\n"); */
-    long v = 0x04030201;
+    u_int32_t v = 0x04030201;
     if (*((unsigned char *)(&v)) == 0x04)
         printf("L_BIG_ENDIAN\n");
     else
