--- leptonlib-1.38/src/Makefile.orig	2006-08-08 00:02:49.000000000 +0200
+++ leptonlib-1.38/src/Makefile	2006-08-27 02:15:29.674206000 +0200
@@ -99,6 +99,8 @@
 OPTIONS =
 CFLAGS =		$(OPTIMIZE) $(OPTIONS)
 LIBRARIAN_SHARED =	gcc -shared
+LDFLAGS =
+LIBADD = -ltiff -ljpeg -lpng -lm
 
 # Libraries differing only in their minor revision numbers
 # are required to have the same interface.  By using
@@ -243,7 +245,7 @@
 
 $(LIB_SHARED)/%.so:
 		$(RM) $@
-		$(LIBRARIAN_SHARED) $(SONAME_OPTION)$(notdir $@).$(MAJOR_REV) -o $@ $<
+		$(LIBRARIAN_SHARED) $(SONAME_OPTION)$(notdir $@).$(MAJOR_REV) $(LDFLAGS) -o $@ $< $(LIBADD)
 	mv $@ $@.$(MAJOR_REV).$(MINOR_REV)
 	cd $(LIB_SHARED); rm $(notdir $@).$(MAJOR_REV); \
 	  ln -s $(notdir $@).$(MAJOR_REV).$(MINOR_REV) $(notdir $@).$(MAJOR_REV)
@@ -275,7 +275,7 @@
 leptlibs:	$(LIB_SHARED)/$(LEPTLIB_SHARED)
 $(LIB_SHARED)/$(LEPTLIB_SHARED):	$(LEPTLIB_C:%.c=$(OBJ_SHARED)/%.o)
 		$(RM) $@
-		$(LIBRARIAN_SHARED) $(SONAME_OPTION)$(notdir $@).$(MAJOR_REV) -o $@ $(LEPTLIB_C:%.c=$(OBJ_SHARED)/%.o)
+		$(LIBRARIAN_SHARED) $(SONAME_OPTION)$(notdir $@).$(MAJOR_REV) $(LDFLAGS) -o $@ $(LEPTLIB_C:%.c=$(OBJ_SHARED)/%.o) $(LIBADD)
 	mv $@ $@.$(MAJOR_REV).$(MINOR_REV)
 	cd $(LIB_SHARED); rm $(notdir $@).$(MAJOR_REV); \
 	  ln -s $(notdir $@).$(MAJOR_REV).$(MINOR_REV) $(notdir $@).$(MAJOR_REV)
